api-catalog-services/src/test/java/org/zowe/apiml/apicatalog/controllers/api/ServicesControllerContainerRetrievalTest.java

```diff
@@ -28,12 +28,12 @@
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
 @ExtendWith(SpringExtension.class)
```

Comment:
not needed

Reply:
Removed by commit https://github.com/zowe/api-layer/pull/4192/commits/09d6415f295f75f3da4f2871eb6687226e66ecfb in the next PR.